core/templates/pages/topic-viewer-page/topic-viewer-page.component.html

```diff
@@ -158,7 +157,7 @@ <h2 class="coming-soon-title">Coming Soon!</h2>
   }
 
   .oppia-topic-viewer-tabs-container .topic-viewer-tabs .topic-viewer-tabs-text .tab-title {
-    font-size: 1em;
+    font-size: medium;
```

Comment:
I don't think the requirement meant medium font-size here.

Medium (Roboto) refers to the font-weight (500).